### PR TITLE
Text improvements in config.sample.php

### PR DESCRIPTION
**Pull request by mmattel**
## Description
If you get an alarm to upgrade an app and you have been logged on via a secondary domain used in ``trusted_domains``, you will get redirected to the doamin used in ``overwrite.cli.url`` because alerts are notifications created by background processes which use by design the url of ``overwrite.cli.url``.

## Related Issue
https://github.com/owncloud/core/issues/31081

## Motivation and Context
Better understanding why you might get redirected

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

@settermjd we might add an additional description in the documentation. Location tbd.

### CODE REVIEW

File: config/config.sample.php [hunk 1]
```diff
@@ -413,6 +415,10 @@
  * are generated within ownCloud using any kind of command line tools (cron or
  * occ). The value should contain the full base URL:
  * ``https://www.example.com/owncloud``
+ * As an example, alerts shown in the browser to upgrade an app are triggered by
+ * a cron background process and therefore uses the url of this key, even the user
```
Comment (phil-davis) [inline config/config.sample.php]: s/even the/even if the/

File: config/config.sample.php [hunk 2]
```diff
@@ -413,6 +415,10 @@
...
+ * has logged on via a different domain defined in key ``trusted_domains``. In case,
+ * the user gets redirected to this url and must relogon.
```
Comment (phil-davis) [inline config/config.sample.php]: Maybe say:
When the user clicks an alert like this they will be redirected to that URL and must logon again.

**Comment by mmattel** [comment]
Thanks @phil-davis for correcting, all suggestions incorporated

**Review by DeepDiver1975 — APPROVED** [review]

**Review by phil-davis — APPROVED** [review]
Text looks good now for my POV.

File: config/config.sample.php [hunk 3]
```diff
@@ -413,6 +415,10 @@
  * are generated within ownCloud using any kind of command line tools (cron or
  * occ). The value should contain the full base URL:
  * ``https://www.example.com/owncloud``
+ * As an example, alerts shown in the browser to upgrade an app are triggered by
+ * a cron background process and therefore uses the url of this key, even if the user
+ * has logged on via a different domain defined in key ``trusted_domains``. When the 
+ * user clicks an alert like this, he will be redirected to that URL and must logon again.
```
Comment (phil-davis) [inline config/config.sample.php]: @settermjd Is there any policy on using "he/she" when referring to "the user"? Do we write randomly 50/50% "he" or "she" or do we write "they"?

(not an issue for this particular PR right now, but would like to know for future writing style)

Comment (mmattel) [reply config/config.sample.php]: I would have said - from my "feeling":
users -> they
user -> he (neutral he, no gender)
But this is my pov and I am not a native speaker :-)

Comment (PVince81) [reply config/config.sample.php]: we already agreed on using the singular "they", so just use "they" everywhere

**Pull request merged**